app/templates/response/add_denial.html

```diff
@@ -55,6 +55,9 @@
             </div>
             <div class="third" hidden>
                 <h4 id="edit-body-header">Edit Email Body</h4>
+                <div id="additional-details-warning" class="alert alert-warning hidden" role="alert">
+                    <strong>Please provide additional details when selecting this reason.</strong>
```

Comment:
Future improvement - we could pull this from the from the database instead of hardcode?

Reply:
I could see us using an "options" JSON column to customize this if we expand on reasons in the future.